functorch/_src/aot_autograd.py

```diff
@@ -1464,6 +1485,7 @@ def functional_call(*args, **kwargs):
             )
         return out
 
+
     def aot_function_simplified(
```

Comment:
imo just delete `aot_function_simplified` and directly call `_create_aot_dispatcher_function`. The more layers of indirection we can remove the better :)

Reply:
Sure